README.rst

```diff
@@ -31,8 +31,38 @@ From GitHub::
 
     $ pip install git+https://github.com/pynamodb/PynamoDB#egg=pynamodb
 
+Upgrading
+=========
+
+.. warning::
+
+    The behavior of 'UnicodeSetAttribute' has changed in backwards-incompatible ways
+    as of the 1.6.0 and 3.0.1 releases of PynamoDB.
+
+When upgrading services that use PynamoDB with tables that contain UnicodeSetAttributes
+with a version < 1.6.0, first deploy version 1.5.4 to prepare the read path for the new
+serialization format.
+
+Once all services that read from the tables have been deployed, then deploy version 2.2.0
+and migrate your data using the provided convenience methods on the Model:
+
+.. code-block:: python
+
+    def get_save_kwargs(item):
+        # any conditional args needed to ensure data does not get overwritten
+        # for example if your item has a `version` attribute
+        {'version__eq': item.version}
+
+    # Re-serialize all UnicodeSetAttributes in the table
+    Model.fix_unicode_set_attributes(get_save_kwargs)
```

Comment:
Suggest elaborating comments a little bit, e.g.:

```
# Re-serialize all UnicodeSetAttributes in the table by scanning all
# objects. See documentation of fix_unicode_set_attributes for
# rate limiting options to avoid exceeding provisioned capacity.
```